api/classes/class-lightweight-api.php

```diff
@@ -413,6 +440,81 @@ public function get_post_payload() {
 		return $payload;
 	}
 
+	/**
+	 * Report Mailchimp data.
```

Comment:
I'd say it's rather querying MC data, this (and the method name) suggests that data is reported _to_ MC.

Reply:
My thinking is that we're fetching data from MC and reporting it to Campaigns data, similar to how we "report" a prompt view or dismissal as an event. But I can see it both ways. feb6b0a renames the method to `get_mailchimp_client_data` instead.